src/formatDate.js

```diff
@@ -50,7 +50,88 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateElements = date.split(fromFormat[fromFormat.length - 1]);
+  const separator = toFormat[toFormat.length - 1];
+  const resultingDate = [];
+
+  let yearFromIndex = 0;
+  let monthFromIndex = 0;
+  let dayFromIndex = 0;
+  let yearToIndex = 0;
+  let monthToIndex = 0;
+  let dayToIndex = 0;
```

Comment:
Instead of storing indexes, you can create an objects with keys: 'DD', 'MM', 'YY', and 'YYYY'